### PR TITLE
Various fixes to workflow dropdown G-UI Components

### PR DESCRIPTION
**Pull request by DavidHollins6**
### 💬 Description
There's a few here so I'll explain them in the comments.

### ✅ Checklist
- [ ] Tests written
- [ ] Browser tested
- [ ] Added to documentation
- [ ] Added to storybook

### CODE REVIEW

File: lib/src/modules/statusIndicatorCircle/statusIndicatorCircle.js [hunk 1]
```diff
@@ -3,14 +3,12 @@ import cx from 'classnames';
...
-    'thick-border': thickBorder,
-    'thin-border': !thickBorder
-  });
+  const className = cx('status-indicator-circle');
+  const boxShadow = `inset 0px 0px 0px ${thickBorder ? '4' : '2'}px ${color}`;
 
   const addedStyles = {
     backgroundColor: solid ? color : 'transparent',
-    borderColor: color
+    boxShadow
```
Comment (DavidHollins6) [inline lib/src/modules/statusIndicatorCircle/statusIndicatorCircle.js:11]: Initially, we had a transition on the border width, this animation looked really choppy. It turns out shadow animations are much cleaner!

File: lib/src/prefabs/dueDateDropdown/DueDateDropdown.js [hunk 2]
```diff
@@ -51,7 +51,10 @@ export function DueDateDropdown({
               <DropdownContent.Footer className="flex">
                 <ButtonSecondaryDanger
                   size={ButtonSecondaryDanger.sizes.sm}
-                  onClick={onRemoveClick}
+                  onClick={async e => {
+                    await onRemoveClick(e);
+                    setShowContent(false);
+                  }}
```
Comment (DavidHollins6) [inline lib/src/prefabs/dueDateDropdown/DueDateDropdown.js:57]: We want to close this dropdown when we have completed any action.

File: lib/src/modules/input/input.scss [hunk 3]
```diff
@@ -39,7 +39,7 @@
 }
 
 .mentions-input--multiLine {
-  @apply h-full;
+  @apply h-full bg-white;
```
Comment (DavidHollins6) [inline lib/src/modules/input/input.scss:42]: The mentions input should have a white background.

**Comment by DavidHollins6** [comment]
[Workflow Mob Test](https://trello.com/c/a5j1E6v1/1478-workflow-mob-test)

**Review by AmeeMorris — APPROVED** [review]

**Pull request merged**